content/docs/handling-events.md

```diff
@@ -29,27 +29,27 @@ is slightly different in React:
 </button>
 ```
 
-Another difference is that you cannot return `false` to prevent default behavior in React. You must call `preventDefault` explicitly. For example, with plain HTML, to prevent the default link behavior of opening a new page, you can write:
+Another difference is that you cannot return `false` to prevent default behavior in React. You must call `preventDefault` explicitly. For example, with plain HTML, to prevent the default form behavior of submitting, you can write:
 
 ```html
-<a href="#" onclick="console.log('The link was clicked.'); return false">
-  Click me
-</a>
+<form onsubmit="console.log('You clicked submit.'); return false">
+  <button type="submit">Submit</button>
+</form>
 ```
 
 In React, this could instead be:
 
-```js{2-5,8}
-function ActionLink() {
-  function handleClick(e) {
+```js{3}
+function Form() {
+  function handleSubmit(e) {
     e.preventDefault();
```

Comment:
Always disliked the `e` naming convention.

Reply:
One benefit over `event` is that there's no built-in called `e`. So you're not going to reference `window.event` accidentally.

Reply:
Maybe! Hopefully lint rules (or TypeScript/Flow) would be sufficient to catch that though.

Reply:
Unfortunately default lint configurations tend to allow it because it's a legit global. But we do forbid it in CRA preset for this reason.